### DISCUSSION TITLE
Is there an upgrade guide for configs?

### DISCUSSION THREAD

**Discussion by grantholle** [yansongda/pay#557] [Q&A] [answered]
I'm trying to update from v2 to v3 and am confused on some of the configuration key changes. Specifically, what v2's `ali_public_key` and `private_key` map to in v3 for alipay.

**Comment by yansongda** [comment, marked as answer]
Because of alipay's recommendation about using certificate,  we have changed some configuration.

`private_key` -> `app_secret_cert` and `app_public_cert_path`

`ali_public_key` -> `alipay_public_cert_path` and `alipay_root_cert_path`

You can check the pay doc via https://pay.yansongda.cn/docs/v3/quick-start/init.html
